classes/various/Matrix.sc

```diff
@@ -44,35 +44,49 @@ Matrix[slot] : Array {
 		});
 		^matrix
 	}
+	*newDFT { arg n;
+		var dft;
+		dft = sqrt(n).reciprocal*super.fill(n, { |r|
```

Comment:
could you please add spaces between the operators? That is only a formatting conventions, but still ...